coderd/coderd.go

```diff
@@ -362,6 +366,14 @@ func New(options *Options) *API {
 		OIDC:   options.OIDCConfig,
 	}
 
+	if options.WorkspaceUsageTracker == nil {
+		options.WorkspaceUsageTracker = workspaceusage.New(options.Database,
+			workspaceusage.WithFlushInterval(workspaceusage.DefaultFlushInterval),
```

Comment:
Consider dropping this in favor of setting the default in `New` (only overridden when using WithFlushInterval)?

Reply:
Fair 👍